mopidy/audio/actor.py

```diff
@@ -740,6 +743,9 @@ def _set_state(self, state):
         :type state: :class:`Gst.State`
         :rtype: :class:`True` if successfull, else :class:`False`
         """
+        if (state < Gst.State.PAUSED):
```

Comment:
The parenthesis are redundant.

Reply:
You'd have thought I would have noticed when I already went and removed the semicolons at the end of lines.